assets/js/hooks/session.js

```diff
@@ -316,7 +320,16 @@ const Session = {
         event.target.closest(`[data-el-outputs-container]`)
       )
     ) {
-      if (cmd && shift && !alt && key === "Enter") {
+      // On macOS, ctrl+alt+- becomes an em-dash, so we check for the code
+      if (event.code === "Minus" && ctrl && alt) {
+        cancelEvent(event);
+        this.goBackNavigationHistory();
```

Comment:
Nitpick:

```suggestion
        this.cursorHistoryGoBack();
```

same in other places.